### ISSUE TITLE
Visiting MISP through SSL returns index.php as text

### ISSUE THREAD

**Issue by hjklz** [MISP/MISP#2510]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Support
| OS version (server)       | CentOS 7
| PHP version               | 5.6
| MISP version / git hash   | 2.4.80

I recently setup MISP again and have run into an issue where when connecting with https, the web ui doesn't load, the code for index.php loads instead. To set up SSL, I copied the non ssl configuration, changed the port to 443 and added the lines about the cert, then re-named the file misp-ssl.conf. Viewing MISP through port 80 yields no issues at all. I am not very familiar with apache's configuration specifics, any assistance is greatly appreciated. Thanks!

### Expected behavior
Load the web UI login page

### Actual behavior
displays the php front controller code (index.php) as text

### Steps to reproduce the behavior
setup ssl on centos 7 following the install doc and visit the server through https

### Logs, screenshots, configuration dump, ...
misp-ssl.conf

```      
<VirtualHost *:443>
        ServerAdmin root@localhost
        ServerName misp.local
        DocumentRoot /var/www/MISP/app/webroot
        <Directory /var/www/MISP/app/webroot>
                Options -Indexes
                AllowOverride all
                                Require all granted
        </Directory>

        <IfModule !mod_php5.c>
            SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1
            DirectoryIndex /index.php index.php
            <FilesMatch \.php$>
                SetHandler "proxy:fcgi://127.0.0.1:9000"
            </FilesMatch>
        </IfModule>

        SSLEngine On
        SSLCertificateFile /etc/ssl/certs/misp.local.crt
        SSLCertificateKeyFile /etc/ssl/certs/misp.local.key

        LogLevel warn
        ErrorLog /var/log/httpd/misp.local_error.log
        CustomLog /var/log/httpd/misp.local_access.log combined
        ServerSignature Off
</VirtualHost>

```

**Comment by adulau** [comment]
I suppose this is related to the overall configuration of Apache and the PHP configuration. Are you sure the PHP module works properly?

**Comment by hjklz** [comment]
I think so, it works fine on http and the rh-php56-php-fpm proxy server is running on localhost:9000. The logs for it also show nothing. Is there something else I should check?

**Comment by hjklz** [comment]
Hmm after more investigation it appears that the only page where I get this error is https://localhost/users/login. The php renders fine through https for every other directory. For example, if I login using http then replace the url with https://localhost/events/index it loads the usual UI via https.

**Comment by hjklz** [comment]
So I found the solution, it was basically me being dumb. 
I had a typo in my php configuration originally. When I changed it, it still gave me the same webpage. What I forgot to do was to clear my browser cache after I reloaded apache.
I cleared browser cache and all worked fine!

So the lesson is to ALWAYS CLEAR CACHE after changing webserver configuration ... which I should've known.
Leaving this here in case other people (or me again!) forget to do the obvious.
